nativeunwind/elfunwindinfo/elfgopclntab.go

```diff
@@ -232,8 +232,8 @@ func getString(data []byte, offset int) []byte {
 const (
 	strategyUnknown = iota
```

Comment:
Can we turn this into a type? It also helps navigating the code with `gopls`.

```suggestion
type strategy int

const (
	strategyUnknown strategy = iota
```